packages/conf-bash/conf-bash.1/opam

```diff
@@ -0,0 +1,13 @@
+opam-version: "2.0"
+maintainer: "code@bnwr.net"
+homepage: "https://www.gnu.org/software/bash/"
+bug-reports: "https://github.com/ocaml/opam-repository/issues"
+license: "GPL-3.0-or-later"
+build: ["bash" "--version"]
+depexts: [
+  ["bash"] {os-family = "bsd"}
```

Comment:
Also isn't zsh default on homebrew now, so bash would be needed there?

Reply:
For the moment macos keeps shipping bash though
```
$ /bin/bash --version
GNU bash, version 3.2.57(1)-release (arm64-apple-darwin22)
Copyright (C) 2007 Free Software Foundation, Inc.
```
I am not sure I would like an unnecessary extra install of bash in that case

Reply:
@avsm the aim is to allow packages that call `bash` explicitly to specify that system dependency.

Reply:
Fine by me.